src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -91,28 +91,25 @@ const noJobsMessage = div({ style: { fontSize: 20, margin: '1rem' } }, [
 
 
 const JobHistory = _.flow(
+  Utils.forwardRefWithName('JobHistory'),
   wrapWorkspace({
     breadcrumbs: props => breadcrumbs.commonPaths.workspaceDashboard(props),
     title: 'Job History', activeTab: 'job history'
-  }),
-  Utils.withCancellationSignal
-)(class JobHistory extends Component {
-  constructor(props) {
-    super(props)
+  })
+)(({ namespace, name, workspace, workspace: { workspace: { bucketName } } }, ref) => {
```

Comment:
We need a `useImperativeHandle` on this `ref` to wire up the `refresh` function to the tab bar.